### DISCUSSION TITLE
Add “.exe” suffix to the Windows compiled executable program for  skywalking-eyes and  skywalking-infra-e2e

### DISCUSSION THREAD

**Discussion by codeglzhang** [apache/skywalking#7844] [General]
In skywalking-infra-e2e, we can use `make build` to get a Windows executable program `skywalking-eyes\bin\windows\license-eye`. However, in the command line of Windows, whether the file is an executable program is determined by whether the file has a specific suffix, such as ".exe". Thus, commands such as  `bin/darwin/license-eye -c test/testdata/.licenserc_for_test_fix.yaml header check` will not be executed correctly under Windows. 
![image](https://user-images.githubusercontent.com/17879307/135402537-93e625f4-d376-4995-b692-344896a18938.png)
After renaming the file `skywalking-eyes\bin\windows\license-eye` to `skywalking-eyes\bin\windows\license-eye.exe`, the command is executed.
![image](https://user-images.githubusercontent.com/17879307/135402746-a394f342-8e98-422c-bd38-c822385bfb31.png)
The same is true for `skywalking-infra-e2e`.

**Comment by kezhenxu94** [comment]
Created issue https://github.com/apache/skywalking/issues/7845 to track the issue @codeglzhang

**Comment by wu-sheng** [comment]
@codeglzhang I adjusted #7845 to set up scope as `skywalking-eyes` only. Because there are various script systems(bash, zsh, etc.) could be used in the end user cases. Supporting skywalking-infra-e2e to run in Windows seems not resolve real issue. 
But for `skywalking-eyes`, this makes sense.
